lib/states/tx/houston.R

```diff
@@ -3,82 +3,147 @@ source("common.R")
 
 # VALIDATION: [YELLOW] The Houston PD's Annual Reports don't list traffic
 # figures, but the stop counts don't appear unreasonable for a city of 2M
-# people. 2018 only has partial data.
+# people. 2018 only has partial data. 
+# NEW DATA UPDATE: Number of stops on 2019 is roughly on par with number
+# of stops in 2017. New data for 2018 is the same as old 2018 data (need to fix). 
```

Comment:
This is taken care of now, right? Or have I *still* somehow not managed to get the right data on the cluster? (Sorry 🤦)

Reply:
Taken care of!